### PR TITLE
feat: dashboard UI refactor

### PR DESCRIPTION
**Pull request by lucas-louis**
# Description

Update dashboard UI pages:

- New navigation bar
- New drive page
- New shared drive page
- New programs page
- New contacts page
- New trash page
- New account page

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [X] I have assigned this PR to myself
- [X] I have added at least 1 reviewer
- [X] I have added the needed labels
- [X] I have linked this PR to an issue
- [X] I have linked this PR to a milestone
- [X] I have linked this PR to a project
- [X] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: src/components/file/detailsFile/DrawerDetailsFile.tsx [hunk 1]
```diff
@@ -10,13 +19,23 @@ import { IPCContact, IPCFile } from 'types/types';
 
 import colors from 'theme/foundations/colors';
 import getFileType from '../../../utils/getFileType';
+import formatDate from '../../../utils/formatDate';
+import formatFileSize from '../../../utils/formatFileSize';
```
Comment (RezaRahemtola) [inline src/components/file/detailsFile/DrawerDetailsFile.tsx:23]:
```suggestion
import getFileType from './utils/getFileType';
import formatDate from './utils/formatDate';
import formatFileSize from './utils/formatFileSize';
```
should work as we have set a `baseSrc` in `tsconfig.json`

File: src/components/navigation/SideBar.tsx [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+	// 							borderLeft: `5px solid ${colors.red[700]}`,
+	// 						}}
+	// 					>
+	// 						{configTab}
+	// 					</Tab>
+	// 				</TabList>
+	// 			</Tabs>
+	// 		</VStack>
+	// 	</VStack>
+	// );
```
Comment (RezaRahemtola) [inline src/components/navigation/SideBar.tsx:265]: you can remove this if it's not used :smile:

File: src/components/cards/ContactCard.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+import { Box, HStack, Text, useBreakpointValue, useDisclosure } from '@chakra-ui/react';
+import { useState } from 'react';
+import { BsFillFilePersonFill } from 'react-icons/bs';
+
+import { IPCContact } from 'types/types';
```
Comment (RezaRahemtola) [inline src/components/cards/ContactCard.tsx:5]:
```suggestion
import type { IPCContact } from 'types/types';
```

File: src/components/cards/FileCard.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
+import { Box, HStack, Text, useBreakpointValue, useDisclosure } from '@chakra-ui/react';
+import { useState } from 'react';
+import { BsFileEarmarkFill } from 'react-icons/bs';
+
+import { IPCFile } from 'types/types';
```
Comment (RezaRahemtola) [inline src/components/cards/FileCard.tsx:5]:
```suggestion
import type { IPCFile } from 'types/types';
```

File: src/components/cards/ContactCard.tsx (hunk 3 above)
Comment (lucas-louis) [reply src/components/cards/ContactCard.tsx:5]: What is the purpose of this?

File: src/components/cards/FileCard.tsx (hunk 4 above)
Comment (lucas-louis) [reply src/components/cards/FileCard.tsx:5]: What is the purpose of this?

**Review by RezaRahemtola — APPROVED** [review]

**Pull request merged**